src/main/java/com/rls/lms/controllers/UserController.java

```diff
@@ -0,0 +1,34 @@
+package com.rls.lms.controllers;
+
+import com.rls.lms.models.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import com.rls.lms.repositories.UserRepository;
+
+@RestController    // This means that this class is a Controller
+@RequestMapping(path="/api") // This means URL's start with /demo (after Application path)
+public class UserController {
+    @Autowired // This means to get the bean called userRepository
+    // Which is auto-generated by Spring, we will use it to handle the data
+    private UserRepository userRepository;
+
+    @GetMapping(path="/add") // Map ONLY GET Requests
+    public @ResponseBody
+    String addNewUser (@RequestParam String name
+            , @RequestParam String email) {
+        // @ResponseBody means the returned String is the response, not a view name
+        // @RequestParam means it is a parameter from the GET or POST request
+
+        User n = new User();
+        n.setName(name);
+        n.setEmail(email);
+        userRepository.save(n);
+        return "Saved";
+    }
+
+    @GetMapping(path="/all")
```

Comment:
**/v1/users** would be better i guess :)

Reply:
I am creating a thread for this discussion.